lib/features/welcome/views/widgets/welcome_button.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:kahootify/color_consts.dart';
+
+class WelcomeButton extends StatelessWidget {
+  final Function onPressed;
+  final String value;
+
+  const WelcomeButton({Key key, this.onPressed, this.value}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: 250,
+      height: 100,
+      child: RaisedButton(
+        color: kBackground1Color,
+        shape: RoundedRectangleBorder(
+            borderRadius: BorderRadius.circular(20.0)),
+        elevation: 50,
```

Comment:
that's a lot of elevation. Are you sure we need all of it?

Reply:
Ready